SYCL/KernelAndProgram/undefined-symbol.cpp

```diff
@@ -1,4 +1,4 @@
-// XFAIL: cuda || hip || (opencl && gpu)
+// XFAIL: cuda || hip
```

Comment:
Could you clarify why this test still fails in Jenkins environment?

Reply:
The test fails on Linux as well. I suggest we investigate the difference between test behavior in Jenkins and GitHub Actions environments.

Reply:
Yes, but I don't understand which test suite is used on jenkins for linux.
Yes, we should investigate. I'll try to figure out.

Reply:
If I understand correctly, drivers in Github Actions docker container for OCL GEN9 more recent, than in the jenkins. 
OCL GEN9 Github Actions have version `22.11.22682`, that matches version from the [uplift PR](https://github.com/intel/llvm/pull/5867/commits/2ffd1d96de814752bf1c023863a5e859fed0d91a) for 'linux_staging'. Can be seen in raw logs from [here](https://github.com/intel/llvm/runs/5661913688?check_suite_focus=true)

Jenkins [have](http://icl-jenkins.sc.intel.com:8080/blue/organizations/jenkins/LLVM-Test-Suite-CI-TMP%2FLLVM-Test-Suite-CI-Linux/detail/LLVM-Test-Suite-CI-Linux/2237/pipeline/156/) version `22.10.22597`.

Anyway, we can't enable test only for specific driver version. Should we disable it for now, while it fails on some driver versions?

Reply:
I reverted driver version in GHA precommit to 22.10.*. 
This patch should go together with https://github.com/intel/llvm/pull/5897.